ush/fv3gfs_regrid_nemsio.sh

```diff
@@ -28,9 +28,9 @@ DATA=${DATA:-$pwd}
 NWPROD=${NWPROD:-$pwd}
 HOMEgfs=${HOMEgfs:-$NWPROD}
 FIX_DIR=${FIX_DIR:-$HOMEgfs/fix}
-FIX_AM=${FIX_AM:-$FIX_DIR/fix_am}
-FIXfv3=${FIXfv3:-$FIX_DIR/fix_fv3_gmted2010}
-REGRID_NEMSIO_EXEC=${REGRID_NEMSIO_EXEC:-$HOMEgfs/exec/regrid_nemsio}
+FIX_AM=${FIX_AM:-$FIX_DIR/am}
```

Comment:
This script can be deleted.  It is no longer in use.

Reply:
We have a separate issue already open to remove all the nemsio stuff. Someone just needs to go through it all.